modules/administration/pages/image-management.adoc

```diff
@@ -470,13 +470,18 @@ However if the certificate path was changed or unavailable, you will need to cal
 /usr/sbin/mgr-package-rpm-certificate-osimage --ca-cert-full-path /root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT
 ----
 
-The RPM package with the certificate is stored in a salt-accessible directory such as `/usr/share/susemanager/salt/images/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm`.
-
-The RPM package with the certificate is provided in the local build host repository `/var/lib/Kiwi/repo`.
+The RPM package with the certificate is stored in a salt-accessible directory such as:
+----
+/usr/share/susemanager/salt/images/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm.
+----
 
+The RPM package with the certificate is provided in the local build host repository:
+----
+/var/lib/Kiwi/repo.
+----
 
 [IMPORTANT]
-.The RPM Package with the {productname} Certificate Must Be Specified in the Build Source
+.Specify the RPM Package with the {productname} Certificate in the Build Source
```

Comment:
We don't put a title on admonitions, because we don't have a proper style implemented for them (yet!). I would move this information in to be the first line in the admon, rather than a title.